peregrine/api.py

```diff
@@ -155,8 +156,7 @@ def health_check():
 @app.route('/_version', methods=['GET'])
 def version():
     dictver = {
-        'version': DICTVERSION,
-        'commit': DICTCOMMIT,
+        'version': pkg_resources.get_distribution("gdcdictionary").version
```

Comment:
this is a breaking API change, since the `commit` is no longer in the response. So if the portal assumes that, things might not work as expected.

I would recommend still having the `commit` field but just make it an empty string, so we don't have to make a breaking change to the API. if you want you could add a note to deprecate it later. unless you're certain that the front-end doesn't expect this.. and even so, if we can avoid the breaking change I'd rather do that